Enterprise-CMCS/managed-care-review
# MR-3175 CHIP only federal authorities list

## Summary
State cannot select an operating authority that is not relevant to CHIP-Only

#### Related issues
https://qmacbis.atlassian.net/browse/MR-3175
https://qmacbis.atlassian.net/browse/MR-3177 - last sprint work, just one additional CHIP copy change requested from stakeholders

#### Test cases covered
- `displays correct form fields for federal authorities with CHIP only contract`
- `displays correct form fields for federal authorities fields with Medicaid contract`
- `renders federal authorities for a medicaid contract`
- `renders federal authorities for a CHIP contract as expected, removing invalid authorities`
- `removes any invalid federal authorities from CHIP submission and submits successfully`

## QA guidance

-See ticket AC.
-  On submission type page you can toggle between CHIP and medicaid population covered - this will impacts federal authorities list on contract details (on 1115 Waiver and Title XXI are listed for CHIP). That will also be displayed options on submission summary.
